src/dao/AbstractContractDAO.js

```diff
@@ -157,6 +157,11 @@ export default class AbstractContractDAO {
     return this._at || this.contract.then(i => i.address)
   }
 
+  async getGasPrice(): Promise<Number> {
+    const gasPrice = await this._web3Provider.getGasPrice()
+    return this._c.fromWei(gasPrice.toNumber())
```

Comment:
can't find implementation of this function
and I have doubts that we should place this function exactly here